behind-the-veil-siteroot/imports/ui/components/message/Conversation.jsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * File Description: Direct Message Conversation Component of Messages Page
+ * File version: 1.0
+ * Contributors: Nishan
+ */
+
+import React, { useRef, useState } from 'react';
+import Card from '../card/Card';
+import ProfilePhoto from '../profilePhoto/profilePhoto';
+import { PaperAirplaneIcon } from '@heroicons/react/24/outline';
+
+export const Conversation = ({ user }) => {
+
+    const [formValue, setFormValue] = useState('');
+    const [messages, setMessages] = useState([]);
+    const conversationRef = useRef(null);
+
+    const sendMessage = (e) => {
+        e.preventDefault();
+        if (formValue.trim() === '') return;
+        setMessages(prevMessages => [...prevMessages, { text: formValue, sender: 'me' }]);
+        setFormValue('');
+        let heightToScroll = conversationRef?.current.scrollHeight + 50
+        setTimeout(() => {
+            conversationRef?.current.scrollTo({ left: 0, top: heightToScroll, behaviour: "smooth" })
+        }, 5)
+
+    };
+
+    return (
+        <div className="flex flex-col fixed top-0 bottom-0 left-0 right-0">
+            <div className='ml-4 w-11/12 message-receiver-name-text border-b-2 pt-3 pb-1 mb-8 pl-6 border-main-blue'>{user.name}
+            </div>
+            <div className="flex-1 overflow-y-auto p-4" ref={conversationRef}>
+                <div>
+                    {user.messages.map((message, index) => (
+                        <div key={index}>
+                            <div className={`${message.sender === 'me' ? 'flex  justify-end' : 'flex'}`}>
+                                <div> {/* wrapping the profilephoto inside this div helps to keep the size consistent when smaller screen */}
```

Comment:
If you're trying to prevent the profile photo from shrinking, you don't need this wrapping div. You can remove it and just add 'shrink-0' to the profile photo's classname

Reply:
oh okay thank you, i fixed it